Update tutorial-template-model-bean.asciidoc (#881)
vaadin/flow-and-components-documentation: documentation/polymer-templates/tutorial-template-model-bean.asciidoc

}
----

> * If you later update the `Person person` bean created in the constructor, the model remains unaffected. The bean values are copied by `setPerson` and the bean is not attached to the model in any way.

Reviewer: ```suggestion
* If you later on update the `Person person` bean created in the constructor, **the model remains unaffected**. The bean values are copied only when the `setPerson` method is called. Thus, the bean is not attached to the model in any way.
```
